microsoft-edge/devtools-guide-chromium/settings/throttling.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Throttling
+description: The Throttling page of DevTools Settings, to control and customize network and CPU speed settings.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 06/12/2025
+---
+<!-- Copyright Sofia Emelianova
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->
+# Throttling
+<!-- https://developer.chrome.com/docs/devtools/settings/throttling -->
+
+In the **Throttling** page of **Customize and control DevTools** (![Customize and control DevTools icon](./throttling-images/customize-and-control-devtools-icon.png)) > **Settings**, you can:
+
+* Calibrate CPU throttling presets to approximate the typical performance of low- and mid-tier mobile devices.
+
+* Set up custom throttling profiles.  You can use your custom throttling profiles to test custom connection speeds in the **Network** tool.<!--, per [Create custom throttling profiles](../network/reference.md#create-custom-throttling-profiles) in _Network features reference_.  todo: after https://github.com/MicrosoftDocs/edge-developer/pull/3405 is merged, link -->
```

Comment:
dependencies strategy?  how to flag this to do after other PR merges?  A solution: 

1. Add note at top of other PR (https://github.com/MicrosoftDocs/edge-developer/pull/3405) to reveal this link. 
1. Merge present PR.
1. Merge "main" into other PR, 
1. In other PR, reveal this link.
1. Merge other PR. 

@captainbrosset

Reply:
Maybe just add a comment to the PR on GitHub? Or combine the 2 PRs into 1 PR?

Reply:
probably keep PR.
It took work to break out the present PR.  in other PR, I made the sections (inline steps) independent of this PR.